atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchManagerImplTest.java

```diff
@@ -40,6 +41,9 @@
     @Mock
     private LockWatchEventCache lockWatchEventCache;
 
+    @Mock
+    private LockWatchValueScopingCache valueScopingCache;
```

Comment:
Probably doing a couple of really basic tests confirming that the methods pass through to the right caches